### PR TITLE
feat: Add simple low-level component example

### PR DESCRIPTION
**Pull request by Chromosomologist**
## Summary

Since we have no examples on directly sending components and using listeners for them, I thought it'd be nice to add an example to the list before I go on exam microhiatus.

It's probably a tad too wordy so any feedback is welcome.

## Checklist

- [ ] If code changes were made, then they have been tested
    - [ ] I have updated the documentation to reflect the changes
    - [x] I have formatted the code properly by running `task lint`
    - [x] I have type-checked the code by running `task pyright`
- [ ] This PR fixes an issue
- [ ] This PR adds something new (e.g. new method or parameters)
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: examples/low_level_components.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+bot = commands.Bot(command_prefix=commands.when_mentioned)
+
+
+# As an alternative to using views, it is possible to use a more low-level approach to components.
+# Firstly, components do not have to be sent as part of a view. Instead, they can be sent as-is.
+# Take special note of the fact that `custom_id`s have been explicitly set on these components.
+
+# The main advantage of this is that listeners are, by nature, persistent. Listeners also do not
+# suffer views' problem of causing a memory leak, where view instances will keep piling up.
```
Comment (onerandomusername) [inline examples/low_level_components.py:16]: tbh we shouldn't mention that views have a memory leak, as that's an implementation detail, and we are working on fixing this issue.

**Review by onerandomusername — CHANGES_REQUESTED** [review]

File: examples/low_level_components.py (hunk 1 above)
Comment (thesadru) [reply examples/low_level_components.py:16]: Or just note that it's a leftover from discord.py and blame it on them. 👍

Comment (shiftinv) [reply examples/low_level_components.py:16]: The increased memory usage of views should be mentioned, but yeah, the fact that views are kept in memory longer than necessary is something that is planned to be fixed.

Comment (Chromosomologist) [reply examples/low_level_components.py]: Alright, changed it ^^

**Review by onerandomusername — APPROVED** [review]
thanks
